integr8ly/workload-web-app
# add monitoring for amq

Create PR to get early feedback. Can be merged if it helps unblock others while I continue working in improve the implementation (need to auto reconnect when a connection is broken)

To verify:

1. Login to an RHMI cluster using oc. You can use https://console-openshift-console.apps.cluster-rhmi-3626.rhmi-3626.example.opentlc.com/ (ping me for credentials)
2. Check out the PR and run `make deploy`. Wait for it to complete and you should see the pod is running.
3. Change the image of the deploymentconfig to "quay.io/integreatly/workload-web-app:dev" and you should see some log messages.
4. Remove the "ENVIRONMENT" env var from the deploymentconfig and you should see debug logs from the pod. You should see logs about received messages.
